plugins/modules/get_certificate.py

```diff
@@ -99,6 +99,14 @@
         - The default value V(false) is B(deprecated) and will change to V(true) in community.crypto 3.0.0.
       type: bool
       version_added: 2.12.0
+    ssl_ctx_options:
+      description:
+        - SSL CTX options (SSL OP flags) to use for the request.
+        - See the L(List of SSL OP Flags,https://wiki.openssl.org/index.php/List_of_SSL_OP_Flags) for more details.
+        - The available SSL CTX options is dependent on the Python and OpenSSL/LibreSSL versions.
+      type: list
+      elements: [ str, int ]
+      version_added: tbd
```

Comment:
Please advise what value to set for version_added.

Reply:
Use 2.21.0, that will be the next minor release.